src/index.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
- [ ] Your app also shows a blank page locally and attached is the error message on the console. It has to do with styles maybe tailwind issues. Kindly check tailwind documentation on how to fix this error.

<img width="955" alt="chrome_HcLFJG0hpN" src="https://user-images.githubusercontent.com/81468675/169604342-c7629ee8-92ea-4e4e-abbb-ae36802cd4ed.png">

Reply:
@julie-ify Please do you have any idea how to fix this?

Reply:
@kosher9 Sorry, I haven't gotten this kind of error before and I have not used tailwind with react. You can request second opinion to see if the Senior code reviewers can help.

Reply:
- [ ] [OPTIONAL] The style error has been fixed but we still have another error in the console that needs to be fixed. You are providing the wrong property type. It expects a string while you provided a number. Kindly fix this.
![Screenshot from 2022-05-21 00-03-26](https://user-images.githubusercontent.com/58692590/169623119-10f77170-d6f8-473c-9442-975d2f7085c6.png)